### PR TITLE
Adjust PV object RBAC  for CSI PODs

### PR DESCRIPTION
**Pull request by humblec**
- [x] PV rbac has been adjusted for provisioner runner
 
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: deploy/examples/common.yaml [hunk 1]
```diff
@@ -19,12 +19,6 @@ rules:
   - apiGroups: [""]
     resources: ["namespaces"]
     verbs: ["get", "list"]
-  - apiGroups: [""]
```
[the remark this answers is not in the document]
Comment (humblec) [reply deploy/examples/common.yaml:22]: Ah :(.. didnt notice that commit ..

[the remark this answers is not in the document]
Comment (humblec) [reply deploy/examples/common.yaml:22]: let me drop the second commit from here..

[the remark this answers is not in the document]
Comment (subhamkrai) [reply deploy/examples/common.yaml]: sure thanks

**Comment by humblec** [comment]
Cc @Madhu-1

**Review by Madhu-1 — CHANGES_REQUESTED** [review]
@humblec please update the commit message and PR title. changes LGTM

**Comment by humblec** [comment]
> @humblec please update the commit message and PR title. changes LGTM

@madhu-1 adjusted .. thanks.. PTAL

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**